server/server.py

```diff
@@ -134,21 +139,21 @@ def submit(sid, data):
   print("answer", sid, data)
   puzzle, answer = [data.get(key) for key in ['puzzle', 'answer']]
   # TODO: error gracefully if no puzzle or answer
-  correct = answer == ANSWERS.get(int(puzzle))
+  correct = answer == ANSWERS.get(str(puzzle))
 
   response = {
     'correct': correct,
   }
   sio.emit('submit_response', response, room=sid)
   if correct:
     table = CLIENTS[sid]
-    GAME_STATE[table][int(puzzle)]['solved'] = True
+    GAME_STATE[table][str(puzzle)]['solved'] = True
     send_game_state(table=table)
 
-  if game_complete(table):
-    GAME_STATE[table]['end_time'] = datetime.now().isoformat()
-    send_game_state(table=table)
-    send_redirect(table, 'FINISH')
+    if game_complete(table):
```

Comment:
this indentation means the following if statement is run inside "if correct" ?

Reply:
yea, when submitting we only need to check if the game is complete if the answer is correct